src/string.rs

```diff
@@ -191,6 +285,23 @@ impl String {
     }
   }
 
+  // Creates a v8::String from a `&'static [u16]`.
+  pub fn new_external_twobyte_static<'s>(
+    scope: &mut HandleScope<'s, ()>,
+    buffer: &'static [u16],
+  ) -> Option<Local<'s, String>> {
+    let buffer_len = buffer.len().try_into().ok()?;
+    unsafe {
+      scope.cast_local(|sd| {
+        v8__String__NewExternalTwoByteStatic(
+          sd.get_isolate_ptr(),
+          buffer.as_ptr() as *const u16,
```

Comment:
```suggestion
          buffer.as_ptr(),
```
The cast is superfluous, I think?